rosa_cli/rosa-get-started-cli.adoc

```diff
@@ -1,11 +1,11 @@
 :_content-type: ASSEMBLY
 include::_attributes/attributes-openshift-dedicated.adoc[]
 [id="rosa-get-started-cli"]
-= Getting started with the rosa CLI
+= Getting started with the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
This is probably a valid case for the full product name since it's a getting started topic, but parentheses can cause rendering issues so be cautious

Reply:
Working fine.